token/program-2022/src/extension/reallocate.rs

```diff
@@ -33,7 +34,7 @@ pub fn process_reallocate(
     let authority_info_data_len = authority_info.data_len();
 
     // check that account is the right type and validate owner
-    let mut current_extension_types = {
+    let (mut current_extension_types, native_amount) = {
```

Comment:
naming: how about `native_token_amount`? it's a bit verbose but less ambiguous (imo) and follows widely-used terminology of `token_amount` by further qualifying it.

Reply:
Sure, done